WebApi/appsettings.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "AllowedHosts": "*",
-  "Token": "5186025303:AAGLxjmNXobY_xdEFMWynZczjOsy75fvY_A"
+  "Token": "1910663617:AAEWtEmyWvGeyz3a10-RTut1Rgh-BiMz2Pg"
```

Comment:
if you have your own, please specify it in appsettings.local.json
so locally you will use yours, but will not commit it